networkmanager/traffic.go

```diff
@@ -276,7 +282,7 @@ func (monitor *trafficMonitoring) deleteAllRules(chain string, rulespec ...strin
 	}
 }
 
-func (monitor *trafficMonitoring) createTrafficChain(chain, rootChain, addresses string) (err error) {
+func (monitor *trafficMonitoring) createTrafficChain(chain, rootChain, addresses string, limit *uint64) (err error) {
```

Comment:
we treat 0 as no limit, it is ok to use `limit uint64` here.

Reply:
Done